.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [pull_request]
+on:
+  push:
```

Comment:
maybe this ca be on PR only?

Reply:
Sure. I wanted to make sure that the test/deploy works on your side.